packages/code/src/LanguageSwitcher/LanguageSwitcher.tsx

```diff
@@ -28,22 +28,13 @@ function isLeafyGreenIcon(element: React.ReactNode) {
   return false;
 }
 
-interface Props extends PopoverProps {
+interface Props {
   language: LanguageOption;
   languageOptions: Array<LanguageOption>;
   onChange: (arg0: LanguageOption) => void;
 }
 
-function LanguageSwitcher({
-  language,
-  languageOptions,
-  onChange,
-  usePortal,
-  portalClassName,
-  portalContainer,
-  scrollContainer,
-  popoverZIndex,
-}: Props) {
+function LanguageSwitcher({ language, languageOptions, onChange }: Props) {
```

Comment:
Haven't checked whether this is new or now, but I noticed that hovering over the menu would also apply the hover styles to the trigger 
![Screenshot 2024-10-14 at 1 27 33 PM](https://github.com/user-attachments/assets/e098fc6d-170e-4653-8e72-715ab052a407)

Reply:
This can also be seen on `Menu` 
<img width="364" alt="Screenshot 2024-10-15 at 12 48 05 PM" src="https://github.com/user-attachments/assets/5e871f1a-3561-4c8f-bf4b-8a741726fcb6">